<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QDialog" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayoutConsole">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayoutConsole">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="consoleLabel">
         <property name="text">
          <string notr="true">&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="consoleLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="consoleClearButton">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Clear console</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../andcoin.qrc">
           <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
         </property>
         <property name="shortcut">
          <string notr="true">Ctrl+L</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTextEdit" name="consoleMessagesWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="tabKeyNavigation" stdset="0">
        <bool>false</bool>
       </property>
       <property name="columnCount" stdset="0">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../andcoin.qrc"/>
 </resources>
 <connections/>
</ui>
